### PR TITLE
Showing List of friends online

### PR DESCRIPTION
**Pull request by Aniruddhmuley2001**
This PR will be solving issue #12 . Here, 3 columns created under headings 'Photo', 'Name', and 'Status'. Also, this has been implemented by creating an object for details of friends, so as to have a dynamic implementation.

@meet59patel Please do review the PR, and suggest the required changes.

### CODE REVIEW

File: src/views/Index.js [hunk 1]
```diff
@@ -34,6 +34,22 @@ import AttendanceCard from "../components/Attendance/AttendanceCard.js";
...
+  name: 'name1',
+  picture: 'https://lh5.googleusercontent.com/-c_qI1qFUiqI/AAAAAAAAAAI/AAAAAAAAAAA/AMZuucnU4NATEYF2mFRLBtU2XOFfLnIgFw/s96-c/photo.jpg',
+  online: true
+},{
+  name: 'name2',
+  picture: 'https://lh5.googleusercontent.com/-c_qI1qFUiqI/AAAAAAAAAAI/AAAAAAAAAAA/AMZuucnU4NATEYF2mFRLBtU2XOFfLnIgFw/s96-c/photo.jpg',
+  online: true
+},{
+  name: 'name3',
+  picture: "https://lh5.googleusercontent.com/-c_qI1qFUiqI/AAAAAAAAAAI/AAAAAAAAAAA/AMZuucnU4NATEYF2mFRLBtU2XOFfLnIgFw/s96-c/photo.jpg",
```
Comment (meet59patel) [inline src/views/Index.js:47]: These profile picture links come with expiry, just like the Google_Token, so these might not even be available the next day we open it. For now just use one of the images in assets folder for this purpose.

Great thing that you used such a dummy object for table data like I had mentioned before 💯

**Review by meet59patel — CHANGES_REQUESTED** [review]

**Comment by meet59patel** [comment]
give @Aniruddhmuley2001 5 points

**Comment by Aniruddhmuley2001** [comment]
@meet59patel Have updated the picture links in the object. But as of now to display the images in the Photo column, have kept a constant image, since I could not figure out a way to dynamically render a local file, without it being stored in a database or cloud (rather getting a URL for the image). But I believe, this won't affect the project since at the end, we will be receiving images as a URL or link, which can easily be integrated.

**Review by meet59patel — APPROVED** [review]
All Good! 💥

**Pull request merged**